README.md

```diff
@@ -120,7 +120,7 @@ DotNET: Instrumentation|Every Wednesday at 10:30 PT|[Google Doc](https://docs.go
 DotNET: SDK|Every Tuesday alternating between 11:00 and 16:00 PT|[Google Doc](https://docs.google.com/document/d/1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C01N3BC2W7Q)|
 Erlang/Elixir: SDK|Every Thursday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1EbBiRjBc_cBf0T_B5OtNRPhbD4jdBrVYJAy8euCDrUI/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C01N75YMZCN)|
 GoLang: SDK|Every Thursday alternating between 10:00 and 15:00 PDT|[Google Doc](https://docs.google.com/document/d/1A63zSWX0x2CyCK_LoNhmQC4rqhLpYXJzXbEPDUQ2n6w/edit#)|[Slack](https://cloud-native.slack.com/archives/C01NPAXACKT)|
-Instrumentation: General|Every Tuesday at 16:00 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
+Instrumentation: General|||[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
```

Comment:
@tedsuo during various conversation it seems the intention was to make this a bi-weekly meeting (which happens once every two weeks) rather than a weekly meeting. Do you remember what was the intention? Any suggestions regarding the frequency?